dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -2769,10 +2769,10 @@
     "RangeInclusivity": {
       "type": "string",
       "enum": [
-        "noneInclusive",
-        "leftInclusive",
-        "rightInclusive",
-        "leftRightInclusive"
+        "NoneInclusive",
```

Comment:
Can I also assume the service code is/will be updated?

Reply:
yes, we have a fix for it.